src/apprt/gtk/App.zig

```diff
@@ -995,6 +995,31 @@ fn loadRuntimeCss(
         unfocused_fill.b,
     });
 
+    if (config.@"window-title-font-family") |font_family| {
+        try writer.print(
+            \\.terminal-window headerbar {{
+            \\  font-family: "{[font_family]s}";
+            \\}}
+            \\
+            \\.inspector-window headerbar {{
+            \\  font-family: "{[font_family]s}";
+            \\}}
+            \\
+            \\.clipboard-confirmation-window headerbar {{
+            \\  font-family: "{[font_family]s}";
+            \\}}
+            \\
+            \\.config-errors-window headerbar {{
```

Comment:
Instead of all this duplication, maybe we should add another class ".window" to every window so that we can target _all windows_ easily too.